### PR TITLE
Hide sensitive info from logs

### PR DESCRIPTION
**Pull request by RosenbergYehuda**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-6966

## Description
We've now ensured that sensitive information directly printed by the request library, especially when debug-mode=true, is properly masked in this PR.

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 1]
```diff
@@ -1757,12 +1753,18 @@ def write(self, msg):
...
+                        try:
+                            text = censor_request_logs(text)
+                        except Exception as e:  # should fail silently
+                            demisto.debug('Failed censoring request logs - {}'.format(str(e)))
+                    if text.startswith('send:'):
+                        try:
+                            self.build_curl(text)
+                        except Exception as e:  # should fail silently
+                            demisto.debug('Failed generating curl - {}'.format(str(e)))
                 demisto.info(text)
```
[the remark this answers is not in the document]
Comment (RosenbergYehuda) [reply Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:1767]: Done.

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py [hunk 2]
```diff
@@ -9554,3 +9554,68 @@ def test_create_clickable_test_wrong_text_value():
...
+@pytest.mark.parametrize("request_log", [
+    ('send: hello\n'),
+    ('header: Authorization\n')
+])
+def test_logger_write__censor_request_logs_has_been_called(mocker, request_log):
+    """
+    Given:
+        A request log that starts with 'send' or 'header' that may contains sensitive data.
+    When:
+        Running logger.write function.
```
Comment (JasBeilin) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py:9607]:
```suggestion
        Running logger.write function when using debug-mode.
```

**Review by JasBeilin — APPROVED** [review]
Nice!

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 3]
```diff
@@ -49,8 +49,8 @@ def __line__():
 ASSETS = "assets"
 EVENTS = "events"
 DATA_TYPES = [EVENTS, ASSETS]
-
-SECRET_REPLACEMENT_STRING = '<XX_REPLACED>'
+MASK = '<XX_REPLACED>'
+SEND_PREFIX = "send: b'"
```
Comment (dantavori) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:53]:
```suggestion
SEND_PREFIX = 'send: b"'
```
image

**Pull request merged**